server/api/jurisdictions.py

```diff
@@ -193,7 +193,9 @@ def min_num_ballots_for_contest(contest: Contest) -> int:
             CvrBallot.query.join(Batch)
             .filter_by(jurisdiction_id=jurisdiction.id)
             .count()
-            if processing and processing["status"] == ProcessingStatus.PROCESSED
+            if (not round_status)
+            and processing
+            and processing["status"] == ProcessingStatus.PROCESSED
```

Comment:
Double checking: Any client-side query invalidation we need to do to make sure we refetch jurisdiction data after launching an audit (or unlaunching an audit)?

Reply:
Good question! Looks like we do that here for launch: https://github.com/votingworks/arlo/blob/jonah/optimize-jurisdictions-endpoint/client/src/components/AuditAdmin/AuditAdminView.tsx#L50

We don't, however, do it for undoing the launch. Added that here: dfdc677c

Reply:
Perfect, thanks for checking/adding!